test/cpp_extensions/open_registration_extension/torch_openreg/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+    "torch", # Needed by setup.py for getting include of PyTorch
+]
+
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "torch_openreg"
+version = "0.0.1"
+description = "A minimal reference implementation of an out-of-tree backend"
+readme = "README.md"
+requires-python = ">=3.9,<3.14"
```

Comment:
Let's remove the higher limit. All of this should work just fine with newer versions